test/module.test.ts

```diff
@@ -0,0 +1,143 @@
+import PutData from './fixture/models/PutData'
+import PostData from './fixture/models/PostData'
+import HeadData from './fixture/models/HeadData'
+import DeleteData from './fixture/models/DeleteData'
+import ErrData from './fixture/models/ErrorData'
+import PatchData from './fixture/models/PatchData'
+import HTTP from '../lib/core/HTTP'
+import OurApiResponse from './fixture/utils/OurApiResponse'
+import GetDataWithParams from './fixture/models/GetDataWithParams'
+import config from './fixture/nuxt.config'
+const { Nuxt, Builder } = require('nuxt-edge')
```

Comment:
Can we use [@nuxt/test-utils](https://github.com/nuxt/test-utils) for tests?  
This module is the future of module testing.

Reply:
Sure, I'll do the refactoring.